src/Uno.UI/Microsoft/UI/Xaml/Controls/PipsPager/PipsPagerTemplateSettings.Properties.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class PipsPagerTemplateSettings : DependencyObject
+	{
+		public IList<object> PipsPagerItems
```

Comment:
The default for this doesn't seem to be null when I try with WinUI 2.7 in C#, but again, the C++ sources doesn't seem to indicate a non-null empty list. 😕